### PR TITLE
cli: adding meta (metadata) commands

### PR DESCRIPTION
**Pull request by temujin9**

### CODE REVIEW

File: cli/meta.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+func runMetaGet(app *types.App, args *docopt.Args, client *controller.Client) error {
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 8, 2, '\t', 0)
+	fmt.Fprintf(w, "KEY\tVALUE\n")
+	for k, v := range app.Meta {
+		fmt.Fprintf(w, "%v\t%v\n", k, v)
+	}
+	w.Flush()
+	return nil
+}
```
Comment (lmars) [inline cli/meta.go]: We have [a function for creating a standard tabwriter](https://github.com/flynn/flynn/blob/2a16a7f46eff77f2501bd3b13b4116537e9a8e48/cli/main.go#L273-L275) and we also have [a convenience function for writing to it](https://github.com/flynn/flynn/blob/2a16a7f46eff77f2501bd3b13b4116537e9a8e48/cli/main.go#L277-L286) (see [here](https://github.com/flynn/flynn/blob/master/cli/ps.go#L34-L46) for example usage).

**Comment by temujin9** [comment]
Okay, this should be ready to go. I don't like the duplication of code between api.UpdateApp and api.UpdateAppMeta, but it does work correctly.

**Comment by titanous** [comment]
The implementation looks good, but it needs tests.

**Comment by temujin9** [comment]
Heard, sorry. Tests coming soon.

**Comment by temujin9** [comment]
Okay, tests have been added.

**Comment by titanous** [comment]
Looks like the CLI part doesn't have any tests, I can write them if you don't want to.

**Comment by temujin9** [comment]
I didn't find any CLI tests for the 'env' command, which I'd been modeling. If you can point me toward those, I'd be glad to replicate for 'meta'.

**Comment by titanous** [comment]
Here are the tests for `flynn env`: https://github.com/flynn/flynn/blob/be2e27311ce485a0709b15ea913a77d1407fb2e2/test/test_cli.go#L335-L344

**Comment by temujin9** [comment]
Tests for `flynn meta` are now ready.

**Comment by titanous** [comment]
LGTM. Thanks!

**Pull request merged**